libs/api/ebpf_api.cpp

```diff
@@ -2866,13 +2880,19 @@ _ebpf_program_load_native(
     ebpf_handle_t* map_handles = nullptr;
     ebpf_handle_t* program_handles = nullptr;
 
-    UNREFERENCED_PARAMETER(attach_type);
-    UNREFERENCED_PARAMETER(execution_type);
-
     if (UuidCreate(&service_name_guid) != RPC_S_OK) {
+        EBPF_LOG_MESSAGE(
+            EBPF_TRACELOG_LEVEL_ERROR,
+            EBPF_TRACELOG_KEYWORD_API,
+            "_ebpf_program_load_native: Create UUID (service name) failed.");
         EBPF_RETURN_RESULT(EBPF_OPERATION_NOT_SUPPORTED);
     }
+
     if (UuidCreate(&provider_module_id) != RPC_S_OK) {
+        EBPF_LOG_MESSAGE(
+            EBPF_TRACELOG_LEVEL_ERROR,
+            EBPF_TRACELOG_KEYWORD_API,
+            "_ebpf_program_load_native: Create UUID (provider module) failed.");
```

Comment:
Can we instead use `EBPF_LOG_MESSAGE_STRING` and also trace `file_name`? That will help in debugging.

Reply:
If by debugging you mean quickly locating the source file, this is already logging the calling function name ('_ebpf_program_load_native'), so that would be easy to search for, right?

To be clear, I don't mind adding the file name as well, just wanted to be sure I understood the purpose.

Reply:
nm.  I was thinking about the wrong filename :-P

Reply:
I meant if multiple native programs / files are being loaded, how to determine for which file is this failure.

Reply:
Yes. Absolutely.  I misunderstood your original comment.